app.js

```diff
@@ -86,11 +86,34 @@ app.post('/sign/connected-contracts', async (req, res) => {
   try {
     const { accountId } = req.body;
     const connectedContracts = await verifiers.near.getconnectedContracts(accountId);
-    const signature = await badger.issueSignedAccessKeyStampMessage({accountId, connectedContracts});
+    const signature = await badger.issueSignedAccountInfoStamp({accountId, connectedContracts});
```

Comment:
I'm not sure if it's safe to freely return an error message on an endpoint that actually doesn't have any limitation, and performs signings. I think someone could try to attack it to try to get the private key through the error message. I think maybe it's a good idea to leave the signature process in another try catch without including the error message.